GEOSldas_App/GEOSldas_HIST.rc

```diff
@@ -206,6 +206,8 @@ EASEv2_M36.LM:           1
 >>>HIST_CATCHCN<<<        'CNNPP'    , 'GridComp'  ,
 >>>HIST_CATCHCN<<<        'CNGPP'    , 'GridComp'  ,
 >>>HIST_CATCHCN<<<        'CNSR'     , 'GridComp'  ,
+>>>HIST_CATCHCNCLM51<<<   'CNAR'     , 'GridComp'  ,
+>>>HIST_CATCHCNCLM51<<<   'CNHR'     , 'GridComp'  ,
```

Comment:
Not sure if 'CNAR' and 'CNHR' would have worked with (the remnants of) CatchCN45.  I changed it to "51" to be safe and for clarity.

Reply:
> Not sure if 'CNAR' and 'CNHR' would have worked with (the remnants of) CatchCN45. I changed it to "51" to be safe and for clarity.

Ok, sounds good.